python/src/tstools/inputFileIO.py

```diff
@@ -199,16 +202,16 @@ def __init__(self):
 
         self.cal = [0,0,0,0,0,0.0]
         self.decYear = 0.0
-        self.offset = [0,0,0]
-        self.deltaV = [0,0,0]
-        self.expMagX1 = [0,0,0]
-        self.expTauX1 = [0,0,0]
-        self.expMagX2 = [0,0,0]
-        self.expTauX2 = [0,0,0]
-        self.expMagX3 = [0,0,0]
-        self.expTauX3 = [0,0,0]
-        self.lnMag = [0,0,0]
-        self.lnTau = [0,0,0]
+        self.offset = np.array([0.,0.,0.])
+        self.deltaV = np.array([0.,0.,0.])
+        self.expMagX1 = np.array([0.,0.,0.])
+        self.expTauX1 = np.array([1e9,1e9,1e9])
+        self.expMagX2 = np.array([0.,0.,0.])
+        self.expTauX2 = np.array([1e9,1e9,1e9])
+        self.expMagX3 = np.array([0.,0.,0.])
+        self.expTauX3 = np.array([1e9,1e9,1e9])
+        self.lnMag = np.array([0.,0.,0.])
+        self.lnTau = np.array([1e9,1e9,1e9])
```

Comment:
Could consider a helper function for initializing this stuff (you do it all over the place):

```python
def _initNPArray(val):
  return np.Array([val, val, val])
```

🤷‍♂